.gitignore

```diff
@@ -1,2 +1 @@
 /target
-Cargo.lock
```

Comment:
Could you bring this back? I would prefer default setting on cargo.

Reply:
I can take a look at this when I get home. I think in general the usual approach is to let the application author commit the cargo.lock, and to *not* specify concrete versions in *libraries* (such as this one). 
https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html

Reply:
here's a direct link to this exact issue in the FAQ - https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries

tl;dr: the cargo.lock is ignored unless the crate is at the end of the dependency chain (which this crate *never* is.

Reply:
Okay, thanks! But I think it might be better for me to generate this lock file with verified commit myself. I will merge this and push it later.

Reply:
There is no reason to keep committing it if is irrelevant then, surely? It's just extra stuff to track in git and inflates the lines-changed count in commits

Reply:
Hi @Restioson, this conversation is already resolved and this pull request is already merged. If you take a look at the commits you'll see that this was file was *not* tracked as part of this pull request.

Reply:
OK :)